mlflow/ml-package-versions.yml

```diff
@@ -914,3 +914,15 @@ litellm:
           "uvicorn",
         ]
     run: pytest tests/litellm
+
+bedrock:
+  package_info:
+    pip_release: "boto3"
+    module_name: "boto3"
+    install_dev: |
```

Comment:
note: Not sure if it's worth testing dev version of boto3. They release almost dairy.

Reply:
not worth it and the likelihood of regressions in main branch is VERY LOW.